client/src/components/Comics.jsx

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+import CryptoJS from "crypto-js";
+import PropTypes from "prop-types"; // Importer PropTypes
+import { useEffect, useState } from "react";
+import "../Styles/Comics.css";
+
+function Comics({ characterId }) {
+  const [comics, setComics] = useState([]);
+
+  useEffect(() => {
+    if (!characterId) return;
+
+    const publicKey = "d48b70dc690340116472afc5b02aa14b";
+    const privateKey = "c09e9c38566391defe2a6c0f74386a57a394e501";
```

Comment:
Don't include private credentials on Github. Use a `.env` file. See the vite documentation: https://vitejs.dev/guide/env-and-mode#env-files